src/lib/layouts/Docs.svelte

```diff
@@ -111,7 +111,7 @@
                         width="130"
                     />
                 </a>
-                <nav class="aw-main-header-nav">
+                <nav class="aw-main-header-nav" aria-label="Docs" >
```

Comment:
This is confusing, as it does not describe the purpose of this navigation itself. Maybe `Top` or `Main` would be better?